### PR TITLE
Lets not fix any file SF generates or updates automatically

### PR DESCRIPTION
**Pull request by robinvdvleuten**
Some files are generated or updated by Symfony. Lets not check for any "coding standards" in these.

### CODE REVIEW

**Comment by robinvdvleuten** [comment]
Ok it seemed to be working on the first attempt but now these files still get fixed. Looks like a CS caching issue, wait a sec before merging this one.

**Comment by robinvdvleuten** [comment]
Ok it's fixed now + ignored the cache directory which drastically improves run time ;)

File: .php_cs.dist [hunk 1]
```diff
@@ -39,5 +39,8 @@ return \PhpCsFixer\Config::create()
     ->setFinder(
         \PhpCsFixer\Finder::create()
             ->in(getcwd())
+            ->exclude('var/cache')
+            ->notPath('var/SymfonyRequirements.php')
```
Comment (ricbra) [inline .php_cs.dist:43]: These files should be removed from your Symfony project and the composer scripts regenerating them should also be removed (`"Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile`). See https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php#L183

File: .php_cs.dist [hunk 2]
```diff
@@ -39,5 +39,8 @@ return \PhpCsFixer\Config::create()
...
+            ->notPath('web/config.php')
```
Comment (ricbra) [inline .php_cs.dist:44]: Sames goes for `web/config.php`.

**Review by ricbra — COMMENTED** [review]
I'm -1 on adding the generated files here.

I'm in doubt on the `var/cache` directory. Could you describe why this needs to be added?

**Comment by othillo** [comment]
I think it would be better to add this to the README: how to use this project with Symfony

**Comment by ricbra** [comment]
Could you update the readme with these instructions @othillo ?

**Comment by othillo** [comment]
@ricbra see https://github.com/simgroep/coding-standards/pull/12

**Comment by ricbra** [comment]
Thanks, closing this in favour of #12

**Pull request closed without merging**